storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/SnowflakeQueryTemplater.java

```diff
@@ -146,22 +150,23 @@ protected String createJoinQuery(
     return writer.toString();
   }
 
-  // TODO: export as decoded csv file
   @Override
   protected List<String> generateExportTableSqlQuery(String resultTable, String stagingPath) {
-    String exportPath = String.format("%s/", stagingPath.replaceAll("/$", ""));
 
     List<String> exportTableSqlQueries = new ArrayList<>();
-    String createStageQuery = String.format("create or replace stage my_stage;");
+    // TODO: s3 integration needs to be configurable: s3_int
+    String createStageQuery =
+        String.format(
+            "create or replace stage my_s3_stage\n"
+                + "  storage_integration = s3_int\n"
```

Comment:
storage integration needs to be configurable

Reply:
commented under this thread https://github.com/Cimpress-MCP/feast/pull/19#discussion_r458180275